docs/src/administration/users.md

```diff
@@ -15,13 +15,13 @@ Any user added to a project or an environment on Platform.sh will need to [regis
 At the project level:
 
 * **Project Administrator** - A project administrator can change settings and execute actions in any environment.
-* **Project Viewer** - A project reader can view all environments within a project but cannot execute any actions on them.
+* **Project Viewer** - A project reader can view all environments within a project through the management console but cannot execute any actions on them.
```

Comment:
There's a slight subtlety here. It's actually not possible to be a project viewer without also having access to at least one environment.

(if you had project `viewer` and access to 0 environments, you would be automatically removed the next time the user list changes, or the next time an environment is deleted - I discovered recently)

Reply:
And another subtlety... it's HTTP API access, which is really 'management console or the CLI or the API'.. so it's probably just best to note something about SSH access in the environment roles below, and leave this.